src/main/java/org/jfrog/bamboo/util/ServerConfigBase.java

```diff
@@ -0,0 +1,43 @@
+package org.jfrog.bamboo.util;
+
+/**
+ * Created by Bar Belity on 03/12/2019.
+ */
+public class ServerConfigBase {
```

Comment:
Can we use the existing **ServerConfig** class instead of adding the new **ServerConfigBase** class?